index.html

```diff
@@ -0,0 +1 @@
+<!DOCTYPE html><html lang="en"><head><meta charset="UTF-8"><meta name="viewport" content="width=device-width, initial-scale=1.0"><link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin="anonymous"><link href="https://fonts.googleapis.com/css2?family=Inter:wght@700&family=Space+Grotesk:wght@700&family=Space+Mono&display=swap" rel="stylesheet"><title>B&O</title><link rel="stylesheet" href="styles.c3c9715a.css"><script src="main.b2b0b56a.js" defer></script></head><body> <header class="header"> <navigation class="navigation header__navigation"> <a class="logo" href="#"><img class="logo__icon" src="Logo.5e498e46.svg" alt="logo"></a> <div class="navigation__buttons"> <a class="phone" href="tel:+1 234 555-55-55"><span class="phone__label">tel:+1 234 555-55-55</span><img class="phone__icon" src="Icon-Phone-call.766c43ee.svg" alt="icon-phone"></a> <a class="burger" href="#"><img class="burger__icon" src="Icon-Burger-menu.7ba546b0.svg" alt="icon-burger"></a> </div> </navigation> <h1 class="title header__title"> Bring joy back to the everyday </h1> </header> <main class="main"> <section class="section main__section"> <article class="sction__article main__section"> <h2 class="title title__secondiary main__title"> Recommended </h2> <img class="section__picture" src="section1.04fdc41c.jpg" alt="phone1"> <h3 class="section__title"> Phone (1) </h3> <p class="section__paragraph"> Designed with instinct, to bring joy back to the everyday. Through the Glyph Interface, a perfected OS and exceptional dual camera. All startlingly fast. </p> <p class="section__price"> $ 499.00 </p> </article> <article class="sction__article main__section"> <img class="section__picture" src="section2.39a2996a.jpg" alt="phone2"> <h3 class="section__title"> Ear (2) </h3> <p class="section__paragraph"> Perfecting the path from artist to ear. So that music sounds just as it was intended. </p> <p class="section__price"> $ 149.00 </p> </article> <article class="sction__article main__section"> <img class="section__picture" src="section3.538deec8.jpg" alt="phone2"> <h3 class="section__title"> Ear (stick) </h3> <p class="section__paragraph"> Tech you can &#x2019; feel. Leaving just space for sound to come to life. Powered by custom sound technology. </p> <p class="section__price"> $ 119.00 </p> </article> </section> </main> </body></html>
```

Comment:
Please format HTML file with the right indentations, now it is unreadable and hard to understand